tests/publishTests.js

```diff
@@ -318,81 +318,68 @@ console.log("\n*** publish.pubImpl ***");
 
 var pubImplFixture = [{
```

Comment:
A small strategy hint comment would help the reader here - something like 'In the cases where commands should not execute, their entry in the structure has been filled with a placeholder string beginning "shouldn't"'